include/z64actor.h

```diff
@@ -532,4 +532,31 @@ typedef enum {
 #define UPDBGCHECKINFO_FLAG_6 (1 << 6) // disable water ripples
 #define UPDBGCHECKINFO_FLAG_7 (1 << 7) // alternate wall check?
 
+typedef enum {
+    /* 0x0 */ NPC_TALK_STATE_IDLE, // NPC not currently talking to player
+    /* 0x1 */ NPC_TALK_STATE_TALKING, // NPC is currently talking to player
+    /* 0x2 */ NPC_TALK_STATE_ACTION, // An NPC-defined action triggered in the conversation
+    /* 0x3 */ NPC_TALK_STATE_ITEM_GIVEN // NPC finished giving an item and text box is done
+} NpcTalkState;
+
+typedef enum {
+    /* 0x0 */ NPC_PLAYER_TRACKING_AUTO_TURN,  // The actor will periodically turn to face the player
```

Comment:
why "periodically" ? afaict once the "turn to player" sequence is done nothing else happens (until the player moves again)?

"periodically" makes me think the npc would also move to face away from the player after some time

Reply:
You're 100% correct. This comment is actually pretty bad in other ways, too: it does not explain that if the player is in front of the actor, normal head/torso tracking is used. Maybe it's best to just refer to `Npc_UpdateAutoTurn` here, it's difficult to concisely explain what this mode does in a short comment

Reply:
Just changing "periodically" to "gradually" would seem fine to me maybe but yeah you can add a reference to the function for more details